ansys/dpf/core/custom_fields_container.py

```diff
@@ -153,16 +192,15 @@ def shell_field(self, timeid=None, complexid=None):
         return self.get_field(label_space)
     
     def beam_field(self, timeid=None, complexid=None):
-        """Returns the requested field with beam element shapes.
-        If a timeid or a complexid (0 for real and 1 for imaginary) are given, 
-        the field returned is a beam field for a given time and/or 
-        complex type. It throws if the number of fields matching the request is 
-        higher than 1.
+        """Retrieve a field with a beam element shape.
+        
+        You can give a time, complex type, or both. If the number of fields 
+        matching the request is higher than one, an exception is raised.
         
         Returns
         -------
         fields: Field
```

Comment:
We should add in the right class descriptions here to allow sphinx to add the links.